doc/reference/configuration/cfg_replication.rst

```diff
@@ -539,19 +539,38 @@
     | Environment variable: TT_ELECTION_TIMEOUT
     | Dynamic: **yes**
 
-..  _cfg_replication-election_fencing_enabled:
+..  _cfg_replication-election_fencing_mode:
 
-..  confval:: election_fencing_enabled
+..  confval:: election_fencing_mode
 
-    Since version :doc:`2.10.0 </release/2.10.0>`.
+    Since version :doc:`2.11.0 </release/2.11.0>`.
```

Comment:
```suggestion
    Since version :doc:`2.11.0 </release/2.11.0>`.
    In earlier Tarantool versions, use :ref:`election_fencing_enabled <cfg_election-election_fencing_enabled_deprecated>` instead.
```
This hasn't been released yet.